### ISSUE TITLE
Feature request: download automatically

### ISSUE THREAD

**Issue by umaar** [alexrozanski/LlamaChat#6]
Any chance we could just select the model, and then LlamaChat downloads the necessary files automatically?

**Comment by matthieu-lapeyre** [comment]
+1 
I have currently downloaded 20GB+ of models, none is working -_-'

**Comment by alexrozanski** [comment]
@umaar Definitely been thinking about this for the publicly available models. This won't be happening for most though because of their distribution terms 👎

@matthieu-lapeyre If there are specific problems could you open another Issue? I wonder if maybe your problems are related to #3 — I have some ideas for this in the works!

**Comment by umaar** [comment]
That's fair, how about a middle ground - include a direct download link to the model file, or if it's required we accept some terms, direct link to the model landing page?

**Comment by alexrozanski** [comment]
yeah, I think at least for GPT4All which publish a direct download link this will work!

**Comment by mignochrono** [comment]
> That's fair, how about a middle ground - include a direct download link to the model file, or if it's required we accept some terms, direct link to the model landing page?

maybe adding some more information on the Readme page, like where to download, what to download, and how to set up the environment to deploy the app. with sections for each one of the supported models(maybe some links).

**Comment by alexrozanski** [comment]
@mignochrono there are some links already in-app but agreed that these could be improved. I think it would be better to improve the UI here because the plan is to add more models, which could get unwieldy if they're all documented in the README.

> how to set up the environment to deploy the app

What do you mean by this exactly? Building from source, or something else?

**Comment by JulianBissekkou** [comment]
Can someone post a short instruction on how to downlaod and install a model that worked for them?
I also had problems getting it working and this would be nice.

Having a quickstart guide on how to get your first chat running would be nice
